benefits/enrollment_switchio/views.py

```diff
@@ -0,0 +1,30 @@
+from django.http import JsonResponse
+from django.views.generic import TemplateView, View
+
+from benefits.core import models, session
+
+
+class IndexView(TemplateView):
```

Comment:
The analog to this view in the `enrollment` app is decorated to [require an eligible session](https://github.com/cal-itp/benefits/blob/main/benefits/enrollment/views.py#L55). Should we have a mixin on this class-based view to preserve this?

Reply:
I need `EligibleSessionRequiredMixin` as well for #2910, and I realize now that it hasn't been written yet. I suggest we have a separate PR that just introduces that mixin for both of our PRs to use.

Reply:
I created #2931 and will work on it now!

Reply:
@lalver1 if you rebase onto `main`, you can add `EligibleSessionRequiredMixin` to this view now

Reply:
Good catch, thanks! I added the `EligibleSessionRequiredMixin` mixin to both views.